#!/usr/bin/env python
PACKAGE = "lwr_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("K_0_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_1_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_2_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_3_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_4_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_5_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)
gen.add("K_6_joint", double_t,    0, "stiffness parameter joint 0", 500,  0, 1000)

exit(gen.generate(PACKAGE, "dynamic_lwr_stiffness", "stiffness_param"))
